cmd/acr/annotate.go

```diff
@@ -302,11 +302,12 @@ func annotateUntaggedManifests(ctx context.Context,
 		}
 		manifestsCount, annotateErr := annotator.Annotate(ctx, manifestsToAnnotate)
 		if annotateErr != nil {
-			return manifestsCount, annotateErr
+			annotatedManifestsCount += manifestsCount
+			return annotatedManifestsCount, annotateErr
```

Comment:
Why do we return a value when there's an error?

Reply:
We would have still annotated a number of manifests successfully, so it seems valid to nonetheless return the number of items that were in fact deleted. I would argue in fact that we shouldn't event exit the annotation look and just keep going but I kept the existing behavior on that one.